es6.md

```diff
@@ -5,6 +5,7 @@ This document specifies the extensions to the core ESTree AST types to support t
 ```js
 extend interface Program {
     sourceType: "script" | "module";
+    body: [ Statement | ModuleDeclaration ];
```

Comment:
And currently they're not allowed at all. We care about backwards for structures themselves, but not for abstract interfaces which just describe constraints (even now there is a lot of places like this).